src/wallet/wallet.h

```diff
@@ -185,15 +185,15 @@ class CAddressBookData
     bool m_change{true};
     std::string m_label;
 public:
-    const std::string& name;
     std::string purpose;
 
-    CAddressBookData() : name(m_label), purpose("unknown") {}
+    CAddressBookData() : purpose("unknown") {}
 
     typedef std::map<std::string, std::string> StringMap;
     StringMap destdata;
 
     bool IsChange() const { return m_change; }
+    const std::string& GetLabel() const { return m_label; }
```

Comment:
```suggestion
    std::string GetLabel() const { return m_label; }
```

To avoid issues related to #17198